src/main/java/jalasoft/com/utils/Utils.java

```diff
@@ -0,0 +1,35 @@
+package jalasoft.com.utils;
+
+import javax.servlet.http.HttpServletRequest;
+
+import jalasoft.com.models.CustomDate;
+import jalasoft.com.models.Status;
+import jalasoft.com.models.Task;
+
+public class Utils {
+
+	private Utils() {
+	}
+
+	static public Task buildTask(HttpServletRequest request) {
+		String title = request.getParameter(Constants.TITLE_PARAMETER_NAME);

```

Comment:
just to follow conventions, please change it to "public static" instead of "static public"